src/catalog.nit

```diff
@@ -235,6 +235,9 @@ class Catalog
 	# Number of warnings and advices
 	var warnings = new Counter[MPackage]
 
+	# Ratio of warnings * 1000 / loc
+	var warnings_permille = new Counter[MPackage]
```

Comment:
I'm not sure the `permille` term is explicit enough in this case, why not name it `warnings_per_kloc` ?